autogpt_platform/backend/backend/blocks/time_blocks.py

```diff
@@ -97,20 +133,29 @@ def __init__(self):
         )
 
     def run(self, input_data: Input, **kwargs) -> BlockOutput:
-        current_date_time = time.strftime("%Y-%m-%d %H:%M:%S")
+        current_date_time = time.strftime(input_data.format)
         yield "date_time", current_date_time
 
 
 class CountdownTimerBlock(Block):
     class Input(BlockSchema):
-        input_message: Any = "timer finished"
-        seconds: Union[int, str] = 0
-        minutes: Union[int, str] = 0
-        hours: Union[int, str] = 0
-        days: Union[int, str] = 0
+        input_message: Any = SchemaField(
+            description="Message to output after the timer finishes",
+            default="timer finished",
+        )
+        seconds: Union[int, str] = SchemaField(
+            description="Duration in seconds", default=0
+        )
+        minutes: Union[int, str] = SchemaField(
+            description="Duration in minutes", default=0
+        )
```

Comment:
wtf is this type

Reply:
I think this code lives before the auto-type conversion is added to the system, and people want to support passing both string and int at the same time.